composition/tests/events.test.ts

```diff
@@ -481,6 +482,61 @@ describe('events Configuration tests', () => {
         ]),
       );
     });
+
+    test('that an error is returned if a NATS subscribe subject references an invalid argument', () => {
```

Comment:
Please add a test for each case that tests against multiple invalid values, multiple undefined values, and a mixture 🙏